generator/osm2meta.hpp

```diff
@@ -101,4 +109,9 @@ class MetadataTagProcessor : private MetadataTagProcessorImpl
     }
     md.Set(mdType, valid);
   }
+
+  std::string FormatFacebook(std::string &v) const;
```

Comment:
```suggestion
  std::string FormatFacebook(std::string const & v) const;
```

will fix your linker error.

Reply:
I tried your solution with `const`, but this didn't help.

I try to call `MetadataTagProcessor` class from `EditableMapObject` class. But `MetadataTagProcessor` is located in "generator" CMake project, while `EditableMapObject` is in "indexer" project. There is no dependencies between those CMake projects. And linked can't find `MetadataTagProcessor` object file. Do you know how to link "indexer" with "generator" lib?

Reply:
Ah ok, you are very close to the solution ) Sorry, I completely forgot about that.
generator library is not used on mobile devices and should not be a dependency of the indexer library (which is used on mobiles).

So the right implementation to reuse these validation methods from both mobile app and from generator would be to put them somewhere into the Editor library.